velox/docs/functions/bitwise.rst

```diff
@@ -42,3 +42,12 @@ Bitwise Functions
 
     Returns the bitwise XOR of ``x`` and ``y`` in 2's complement representation.
 
+.. function:: bit_count(x, bits) -> bigint
```

Comment:
I forgot to mention that functions in the docs appear in alphabetical order. Would you move this function to preserve that order?